Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -356,6 +356,10 @@ public enum DefaultFunctionOpacityOptions {
 If verification fails, report a detailed counterexample for the first failing assertion (experimental).".TrimStart()) {
   };
 
+  public static readonly Option<string> BackendModuleName = new("--module-name",
```

Comment:
The whole reason of using `module-name` was to support a single flag across backends. If anything, based on our discussion you managed to convince me to use a single-name.
```
Shubham Chaturvedi
Yeah. I think I’ll go with the above approach of go-module-name, python-module-namefor now since it works with full and split doo files


Robin Salkeld
Hmm, I'm not loving that when there's an alternative that wouldn't duplicate all those options. But perhaps your doc will convince me :slightly_smiling_face:
```

I was always for `go-module-name`, my only concern was doing repetitive things like `dafny translate go --go-module` , `translate python --python-module-name` . Although some wasted efforts, I'm happy that the simpler solution works.